webpack.js

```diff
@@ -17,6 +17,7 @@ webpackConfig.module.rules = Object.values(webpackRules)
 webpackConfig.entry = {
 	main: path.resolve(path.join('src', 'main.js')),
 	settings: path.resolve(path.join('src', 'settings.js')),
+	adminSettings: path.resolve(path.join('src', 'adminSettings.js')),
```

Comment:
Not sure if there are any guidelines, but I'd prefer dashed names
```suggestion
	'admin-settings': path.resolve(path.join('src', 'adminSettings.js')),
```